editor/editor_resource_picker.cpp

```diff
@@ -504,6 +505,11 @@ void EditorResourcePicker::set_create_options(Object *p_menu_node) {
 			int id = TYPE_BASE_ID + idx;
 			edit_menu->add_icon_item(icon, vformat(TTR("New %s"), t), id);
 
+			HashMap<String, DocData::ClassDoc>::Iterator class_doc = EditorHelp::get_doc_data()->class_list.find(t);
+			if (class_doc) {
+				edit_menu->set_item_tooltip(-1, class_doc->value.brief_description);
```

Comment:
Needs to go through `DTR` for translations, no?
```suggestion
				edit_menu->set_item_tooltip(-1, DTR(class_doc->value.brief_description));
```